api/v1alpha1/alluxiocluster_types.go

```diff
@@ -257,14 +257,24 @@ type AlluxioMonitorSpec struct {
 type EtcdSpec struct {
 	Enabled      *bool             `json:"enabled" yaml:"enabled"`
 	Auth         EtcdAuthSpec      `json:"auth,omitempty" yaml:"auth,omitempty"`
-	Image        EtcdImageSpec     `json:"image,omitempty" yaml:"image,omitempty"`
```

Comment:
so excluding this meaning? we don't specify a etcd image when using k8s operator?

Reply:
It was incorrect. Fixed.

Reply:
indentation format?

Reply:
Separating the required field with the rest makes the indentation different.